napalm/eos/eos.py

```diff
@@ -1740,6 +1753,12 @@ def _parse_per_peer_bgp_detail(peer_output):
                 item["local_address"] = napalm.base.helpers.convert(
                     napalm.base.helpers.ip, item["local_address"]
                 )
+                # Get all the advertised prefixes
+                item["advertised_prefix_count"] += item["advertised_ipv6_prefix_count"]
+                item["received_prefix_count"] += item["received_ipv6_prefix_count"]
```

Comment:
Why do you mix ipv4 and ipv6 prefixes in the same counter ?
Shouldn't be ipv6 prefixes counter added to its own counter ?

Reply:
the other devices share the naming so adding a separate counter for EOS will also mean adding for the other devices.

Reply:
Ok fair enough.